README.md

```diff
@@ -1,20 +1,64 @@
+
+MMODA is developed joinly by [UNIGE Department of Astronomy](https://www.unige.ch/sciences/astro/en/), [EPFL LASTRO Laboratory](https://www.epfl.ch/labs/lastro), [UniParis APC](https://apc.u-paris.fr/APC_CS/en), [Kyiv Academic University](https://kau.org.ua/en), with some support of [SDSC EPFL](https://www.epfl.ch/research/domains/sdsc/).
+
+<a  href="/mmoda/"
+    title="Home"> <img
+    src="https://www.astro.unige.ch/mmoda/sites/all/themes/bootstrap_mmoda/logo.png" alt="Home"
+    width="100px" /></a>
+<a  href="https://www.unige.ch/sciences/astro/en/"
+    title="Departement of Astronomy - university of Geneva"><img
+    src="https://www.astro.unige.ch/mmoda/sites/all/themes/bootstrap_mmoda/logo-fac-sciences.png" alt="Departement of Astronomy - university of Geneva" 
+    width="100px" /></a>
+<a  href="https://www.isdc.unige.ch/integral/"
+    title="The INTErnational Gamma-Ray Astrophysics Laboratory - INTEGRAL"><img
+    src="https://www.astro.unige.ch/mmoda/sites/all/themes/bootstrap_mmoda/logo-isdc.png" alt="The INTErnational Gamma-Ray Astrophysics Laboratory - INTEGRAL" 
+    width="100px" /></a>
+<a  href="https://www.epfl.ch/labs/lastro"
+    title="Laboratory of Astrophysics (LASTRO) - EPFL"><img
+    src="https://www.astro.unige.ch/mmoda/sites/all/themes/bootstrap_mmoda/logo-epfl.png" alt="Laboratory of Astrophysics (LASTRO) - EPFL"
+    width="100px"/></a>
+<a href="https://apc.u-paris.fr/APC_CS/en" target="_blank"
+   title="Laboratoire AstroParticule et Cosmologie (APC)"><img
+   src="https://www.astro.unige.ch/mmoda/sites/all/themes/bootstrap_mmoda/logo-apc.png" alt="Laboratoire AstroParticule et Cosmologie (APC)"
+   width="30px"/></a>
+<a href="https://kau.org.ua/en" target="_blank"
+   title="Kyiv Academic University (KAU)"><img
+   src="https://www.astro.unige.ch/mmoda/sites/all/themes/bootstrap_mmoda/logo-kau.png" alt="Kyiv Academic University (KAU)" 
+   width="50px"/></a>
+      
+
+## Changes in MMODA 22.07.0000
+
+`22.07.0000` is a major release, adding several new major features, and improving user experience.
+
+## New instruments and integration
+
+* Added [Gravitatitional Wave analysis](https://www.astro.unige.ch/mmoda/help/mmoda/gravitational-wave-analysis)
+* Added [DESI LegacySurvey analysis](https://www.astro.unige.ch/mmoda/help/mmoda/legacy-survey)
+* Added possibility to open a new [RenkuLab](https://renkulab.io/) session
 
 ## User experience improvements
 
-* Some improvements in error reporting in frontend an API.
-* More clear note in the email about expiration time of the URL in the cases when these URLs are temporary.
-* When the URLs provided in job email and in the frontend are persistent they are more consistent: parameters in the URL are converted to default formats before being inserted to the URL (e.g. time will be always represented as ISOT).
+* Some improvements in error reporting in frontend and API
```

Comment:
I would mention also the new error reports within emails

Reply:
they are not added yet to the release, since they are not verified enough in live staging. If we verify we can add. For now - no.